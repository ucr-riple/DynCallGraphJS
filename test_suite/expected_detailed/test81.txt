{
    'global (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test81.js:1:1:22:5)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test81.js:1:2:22:2)'
    ],
    'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test81.js:10:3:19:5)': [
        'defineProperty (Native)(/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test81.js:10:3:19:5)'
    ],
    'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test81.js:20:11:20:14)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test81.js:11:10:13:6)'
    ],
    'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test81.js:21:3:21:6)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test81.js:7:16:9:4)'
    ]
}