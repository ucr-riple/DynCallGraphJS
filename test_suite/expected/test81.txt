{ '(test81.js:1:1:22:5)':
   [ '(test81.js:1:2:22:2)' ],
  '(test81.js:10:3:19:5)': [ 'defineProperty (Native)' ],
  '(test81.js:20:11:20:14)':
   [ '(test81.js:11:10:13:6)' ],
  '(test81.js:21:3:21:6)':
   [ '(test81.js:7:16:9:4)' ] }